docs/source/conf.py

```diff
@@ -123,7 +148,30 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_rtd_theme'
+html_theme = 'sphinx_book_theme'
+html_title = ''
+
+html_context = {
+    'github_user': 'NCAR',
+    'github_repo': 'pop-tools',
+    'github_version': 'master',
+    'doc_path': 'docs',
+}
+html_theme_options = dict(
+    # analytics_id=''  this is configured in rtfd.io
+    # canonical_url="",
+    repository_url='https://github.com/NCAR/pop-tools',
+    repository_branch='master',
+    path_to_docs='docs',
+    use_edit_page_button=True,
+    use_repository_button=True,
+    use_issues_button=True,
+    home_page_in_toc=False,
+    extra_navbar='',
+    navbar_footer_text='',
+    extra_footer="""Theme by the <a href="https://ebp.jupyterbook.org">Executable Book Project</a>""",
+)
+
 html_logo = 'images/nsf.png'
```

Comment:
@mnlevy1981, does POP have a logo we could use here? If so, could you point me to it when you get a moment?

Reply:
I don't think there is one, but I've asked one of the LANL developers to be sure. I'll let you know if I hear anything back.

Reply:
Thanks!